app/repositories/application/OnlineTestRepository.scala

```diff
@@ -163,6 +165,24 @@ class OnlineTestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
     selectRandom(query).map(_.map(bsonDocToExpiringOnlineTest))
   }
 
+  def nextTestForReminder(reminder: ReminderNotice): Future[Option[NotificationExpiringOnlineTest]] = {
+    val query = BSONDocument("$and" -> BSONArray(
+      BSONDocument(
+        "applicationStatus" -> ApplicationStatus.PHASE1_TESTS
+      ),
+      BSONDocument(
+        "testGroups.PHASE1.expirationDate" -> BSONDocument(
+          "$lte" -> dateTime.nowLocalTimeZone.plusHours(reminder.hoursBeforeReminder)) // Serialises to UTC.
+      ),
+      BSONDocument("$and" -> BSONArray(
+        BSONDocument("progress-status.PHASE1_TESTS_COMPLETED" -> BSONDocument("$ne" -> true)),
```

Comment:
This should use the case object's from the ProgressStatuses class please, rather than strings